tests/testthat/test-forge-recipe.R

```diff
@@ -2,70 +2,125 @@ context("test-forge-recipe")
 
 library(recipes)
 
+sparse_bp <- default_recipe_blueprint(composition = "dgCMatrix")
+matrix_bp <- default_recipe_blueprint(composition = "matrix")
```

Comment:
I know in other tidymodels packages we use this space to create objects up front, but I really prefer to keep each test as self contained as possible. I know it is extra work, but could you please just recreate this object in each test that requires it?